docs/dev/adding-new-platform.md

```diff
@@ -0,0 +1,158 @@
+How to add a new platform to OpenShift
+======================================
+
+This document describes the process for adding a new platform to OpenShift in stages. Because there are many components, the process is defined in terms of incremental safe spots that can be built up over time and delivered individually. It is *impossible* to all a new platform to OpenShift all in one chunk.
+
+The general order of adding a new platform for OpenShift is:
+
+### Enable core platform
+
+1. **Boot** - Ensure RH CoreOS boots on the desired platform, that Ignition works, and that you have VM / machine images to test with
```

Comment:
I'd also note here that for new cloud platforms, Ignition may need support upstream.  For example here's a PR for a non-top-tier cloud: https://github.com/coreos/ignition/pull/667